Content/README.md

```diff
@@ -0,0 +1,49 @@
+# SAFE Template
+
+This template can be used to generate a full-stack web application using the [SAFE Stack](https://safe-stack.github.io/).
+
+## Install pre-requisites
+
+You'll need to install the following pre-requisites in order to build SAFE applications
+
+* [FAKE 5](https://fake.build/) installed as a [global tool](https://fake.build/fake-gettingstarted.html#Install-FAKE)
+* The [Yarn](https://yarnpkg.com/lang/en/docs/install/) package manager (you an also use `npm` but the usage of `yarn` is encouraged).
+* [Node LTS](https://nodejs.org/en/download/) installed for the front end components.
+* If you're running on OSX or Linux, you'll also need to install [Mono](https://www.mono-project.com/docs/getting-started/install/).
+
+## Work with the application
+
+To concurrently run the server and the client components in watch mode use the following command:
+
+```bash
+fake build -t Run
+```
+
+//#if (deploy == "docker")
+To build your application as a docker container run the following command:
+
+```bash
+fake build -t Docker
+```
+//#endif
+//#if (deploy == "azure")
+To deploy your application as an Azure Web App run the following command:
+
+```bash
+fake build -t AppService
+```
+//#endif
+
+## SAFE Stack Documentation
+
+You will find more documentation about the used F# components at the following places:
+
+* [Saturn](https://saturnframework.org/docs/)
+* [Fable](https://fable.io/docs/)
+* [Elmish](https://elmish.github.io/elmish/)
+
+If you want to know more about the full Azure Stack and all of it's components (including Azure) visit the official [SAFE documentation](https://safe-stack.github.io/docs/).
+
+## Troubleshooting
```

Comment:
I think you can remove this section - it's something that will be fixed (if it's not already fixed) relatively soon.

Reply:
Do you mean the Troubleshooting paragraph  or the paragraph before that and the Troubleshooting paragraph?

Reply:
I think I saw newest SDK warning about that when installing a global tool, at least on a bare alpine-sdk2.1 docker image

Reply:
Ok I'll just leave it then for now.